Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -272,155 +644,452 @@ def fetch_subscribed_services(client, method, base_url, token):
+        if last_update is None:
+            demisto.error("[get-modified-remote-data] last_update is None after parsing")
+            return GetModifiedRemoteDataResponse([])
 
-    alerts = []
-    for alert in all_alerts:
-        current_timestamp = alert.get("created_at")
-        if current_timestamp == prev_timestamp:
-            alerts.append(alert)
+        if last_update.tzinfo is None:
+            last_update = last_update.replace(tzinfo=pytz.UTC)
         else:
-            alert_count += timestamp_count[prev_timestamp]
-            prev_timestamp = current_timestamp
+            last_update = last_update.astimezone(pytz.UTC)
 
-            if alert_count + timestamp_count[current_timestamp] <= max_fetch and current_timestamp != last_timestamp:
-                alerts.append(alert)
-            else:
-                break
-
-    del all_alerts
-    del timestamp_count
-
-    incidents = []
-
-    if alerts:
-        timestamp_str = alerts[-1].get("created_at")
-        original_datetime = datetime.strptime(timestamp_str, "%Y-%m-%dT%H:%M:%S.%fZ")
-        updated_datetime = original_datetime + timedelta(microseconds=1000)
-        latest_created_time = updated_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
-        events = format_incidents(alerts, hide_cvv_expiry)
-
-        for event in events:
-            inci = {
-                "name": event.get("name"),
-                "severity": event.get("severity"),
-                "rawJSON": json.dumps(event),
-                "alert_group_id": event.get("alert_group_id"),
-                "event_id": event.get("event_id"),
-                "keyword": event.get("keyword"),
-                "created": event.get("created_at"),
-            }
-            incidents.append(inci)
-        next_run = {"event_pull_start_date": latest_created_time}
 
-        return incidents, next_run
+    except Exception as e:
+        demisto.error(f"[get-modified-remote-data] Error parsing last_update: {e}")
+        return GetModifiedRemoteDataResponse([])
+
+    services = get_fetch_service_list(client, incident_collections, url, token)
+    severities = get_fetch_severities(incident_severity)
+
+    input_params = {
+        "order_by": args.get('order_by', "asc"),
+        "skip": 0,
+        "limit": MAX_ALERTS,
+        "take": MAX_ALERTS,
+        "url": url,
+        "api_key": token,
+        "hce": hide_cvv_expiry,
+        "services": services or [],
+        "severity": severities or [],
+        "gte": last_update.isoformat(),
+        "lte": datetime.utcnow().replace(tzinfo=pytz.UTC).isoformat()
+    }
+
+    ids = client.get_ids_with_retry(
+        service=services,
+        input_params=input_params,
+        is_update=True
+    )
+
+    if isinstance(ids, list):
+        return GetModifiedRemoteDataResponse(ids)
     else:
-        return [], {"event_pull_start_date": latest_created_time}
+        demisto.error("[get-modified-remote-data] Invalid response format")
```

Comment:
use `return_error` or raise excpetion
as this fill let server know that the previos mirror was failed